Eth-Global-Guide.md

```diff
@@ -0,0 +1,32 @@
+# Intro to Zetachain
+
+## Introduction
+
+- What is ZetaChain?
+- Why build dApps on Zeta ? (Key features) 
+- What is ZETA token ?
+
+## Build with Zeta
+- Start building dapps ( Refer to the first half -  https://www.zetachain.com/docs/developers/overview/) 
+- Using Zetachain Smart Contracts (conceptual explanation)
+> - Omnichain Contracts
+> - Cross Chain Messaging (explanation 
+- Sending and Depositing Tokens
+- Smart Contract Template
+
+## Smart Contracts
+- Create a simple Omnichain Contract
+- Create a simple Cross Chain Messaging Contract
+
+
+## FullStack tutorial 
+- Bitcoin Frontend (Omnichain)
```

Comment:
The [current Bitcoin  `index.html` tutorial](https://www.zetachain.com/docs/developers/omnichain/tutorials/bitcoin-frontend/) might be a bit too simplistic. Maybe in this set of guides we skip the frontend for now and focus on smart contract development.

Reply:
Makes sense ! maybe we can add this as a hyperlink under Omnichain contract and redirect towards the doc directly !